Solutions/IronNet IronDefense/Data/Solution_IronNetIronDefense.json

```diff
@@ -1,21 +1,29 @@
 {
   "Name": "IronNet IronDefense",
-  "Author": "Microsoft - support@microsoft.com",
+  "Author": "Ecosystems & Integrations - support@ironnet.com",
   "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/IronNet%20IronDefense/Workbooks/Images/Logos/IronNet.svg\"width=\"75px\"height=\"75px\">",
-  "Description": "The [IronNet Collective Defense](https://www.ironnet.com/what-is-collective-defense) solution enables ingestion of IronDefense alerts, events, and IronDome notifications into Microsoft Sentinel, enabling Microsoft Sentinel to utilize IronDefense's behavioral analytics and the IronDome community to quickly identify threats in your enterprise network.\n\n**Underlying Microsoft Technologies used:**\n\nThis solution takes a dependency on the following technologies, and some of these dependencies either may be in [Preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) state or might result in additional ingestion or operational costs:\n a. [Agent-based log collection (CEF over Syslog)](https://docs.microsoft.com/azure/sentinel/connect-common-event-format)",
-  "Data Connectors": [
-    "DataConnectors/IronNetIronDefense.json"
+  "Description": "The [IronNet Collective Defense](https://www.ironnet.com/what-is-collective-defense) solution enables ingestion of IronDefense alerts, events, and IronDome notifications into Microsoft Sentinel, enabling Microsoft Sentinel to utilize IronDefense's behavioral analytics and the IronDome community to quickly identify threats in your enterprise network.\n\n**Underlying Microsoft Technologies used:**\n\nThis solution takes a dependency on the following technologies, and some of these dependencies either may be in [Preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) state or might result in additional ingestion or operational costs:\n\n a. [Agent-based log collection (CEF over Syslog)](https://docs.microsoft.com/azure/sentinel/connect-common-event-format)",
+  "Workbooks": [
+    "Workbooks/IronDefenseAlertDashboard.json",
+    "Workbooks/IronDefenseAlertDetails.json"
+  ],
+  "WorkbookDescription": [
+     "Dashboard to view IronDefense alerts",
+     "View IronDefense alert details from the IronDefense Alerts dashboard"
```

Comment:
Please this whole parameter of workbook description, it should not be included in the input file

Reply:
Hello @ling-tx please work on this change and repackage the solution again

Reply:
I remove the workbook description and repackage again